### DISCUSSION TITLE
Run commands without editing files

### DISCUSSION THREAD

**Discussion by JorgenRingen** [JetBrains/ideavim#754] [Q&A] [answered]
I've mapped a lot of idea-actions to the leader, but to be able to execute them, I have to open a file in normal mode. Is there any way to run "normal mode" commands from anywhere inside intelilj, like when all tabs are closed for example?

Command example:
`nmap <leader>px :action CloseProject<cr>`

**Comment by citizenmatt** [comment, marked as answer]
Unfortunately not. Vim is an editor, and IdeaVim modifies IntelliJ's editor to behave like Vim. As such, it requires an editor context in order to execute commands.
